get_all_manifests.sh

```diff
@@ -14,9 +14,9 @@ declare -A COMPONENT_MANIFESTS=(
     ["odh-notebook-controller"]="opendatahub-io:kubeflow:v1.7-branch:components/odh-notebook-controller/config:odh-notebook-controller/odh-notebook-controller"
     ["notebooks"]="opendatahub-io:notebooks:main:manifests:notebooks"
     ["trustyai"]="trustyai-explainability:trustyai-service-operator:release/1.10.2:config:trustyai-service-operator"
-    ["model-mesh"]="opendatahub-io:modelmesh-serving:release-0.11.0:config:model-mesh"
-    ["odh-model-controller"]="opendatahub-io:odh-model-controller:release-0.11.0:config:odh-model-controller"
-    ["kserve"]="opendatahub-io:kserve:release-v0.11.0:config:kserve"
+    ["model-mesh"]="opendatahub-io:modelmesh-serving:release-0.11.1:config:model-mesh"
```

Comment:
The version bump should be (if not a separate PR) a separate commit IMO

Reply:
I will revert this change, to make it easier to cherry-pick this PR to downstream.

Reply:
Reverted.